fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -708,6 +708,16 @@ private ColumnDef getColumnDef(StarRocksParser.ColumnDescContext context) {
         } else if (context.NULL() != null) {
             isAllowNull = true;
         }
+        Boolean isAutoIncrement = null;
+        if (context.AUTO_INCREMENT() != null) {
+            isAutoIncrement = true;
+        }
+        if (isAutoIncrement != null && isAllowNull != null && isAllowNull == true) {
```

Comment:
You can output the column name in the exception.

Reply:
have fixed it